STANDARD.md

```diff
@@ -12,9 +12,9 @@ interpreted as described in [RFC 2119](http://tools.ietf.org/html/rfc2119).
 
 All `keywords` in this standard are case-sensitive.
 
-The naming *(float)* without a closer specification is used if the implementor
+The naming *(floatX)* without a closer specification is used if the implementor
 can choose which kind of floating point precision shall be used.
```

Comment:
Here, just for clarification, I would add examples:
```
can choose which kind of floating point precision shall be used (e.g. float32, float64, float128, etc.)
```

Reply:
Good point, makes it less abstract.

Reply:
It would probably make sense to then rename all the `double` namings to `float64` explicitly as we already do it for the Fortran equivalents.
C++ is not to sane in defining it's "double" definition and we really do mean float64 anyway where used.